Neos.ContentRepository.Core/Classes/Feature/WorkspaceModification/Event/WorkspaceOwnerWasChanged.php

```diff
@@ -0,0 +1,35 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Neos\ContentRepository\Core\Feature\WorkspaceModification\Event;
+
+use Neos\ContentRepository\Core\SharedModel\Workspace\WorkspaceName;
+use Neos\ContentRepository\Core\EventStore\EventInterface;
+
+/**
+ * Event triggered to indicate that the owner of a workspace has changed.
+ *
+ * @api events are the persistence-API of the content repository
+ */
+final class WorkspaceOwnerWasChanged implements EventInterface
+{
+    public function __construct(
+        public readonly WorkspaceName $workspaceName,
+        public readonly ?string $workspaceOwner,
```

Comment:
should we change this to $newWorkspaceOwner?

and explain what the semantics are when setting this to NULL.

Reply:
Done.